src/redux/categories/categories.js

```diff
@@ -0,0 +1,23 @@
+export const categoriesState = [];
+
+const ADD_CATEGORY = 'ADD_CATEGORY';
+const REMOVE_CATEGORY = 'REMOVE_CATEGORY';
+
+export const categoriesReducer = (categoriesState, action) => {
+  switch (action.type) {
+    case ADD_CATEGORY:
+      return [...categoriesState, action.category];
+    case REMOVE_CATEGORY:
+      return [...categoriesState].filter((category) => category.id !== action.id);
+    default:
+      return categoriesState;
+  }
+};
```

Comment:
- [ ] From the category actions types and creators, you have to implement the following.

`CHECK_STATUS` as an action type has to be implemented.

- Sets an initial state with an empty array.
- Creates and Exports **Action Creator**. (`checkStatus` function)

Reply:
Hi @Emmyn5600 
thanks for the review and for the comments, do you mean here that i have to add CHECK_STATUS  as an action ?
and if you can help more what is it for please